mapping/collection.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+module Mapping
+  class AdminSet
+    require 'csv'
+    require 'yaml'
+
+    extend Extensions::BasicValueHandler
+
+    ##
+    # Convert data in a csv file to hash in YAML format
+    #
+    # @param[String] value - the path to the csv file
+    # return - [Hash] of YAML
+    def csv_to_yaml(value)
+      data = CSV.read (value, :headers => true).map(&:to_hash)
+      data.to_yaml
+    end
+
+  ##
+  # Get ownerId from object_properties and parse DSpace owning collection name from lookup table 
+  def owner_collection_name
```

Comment:
This method needs to have the method signature so that the owner_id value is passed into it.

`def owner_collection_name(value, *args)`